.github/workflows/supported_storages_compatibility_check.yaml

```diff
@@ -0,0 +1,728 @@
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_postgresql_13_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-postgresql-14-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: postgres:14.9-alpine
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:postgresql://localhost:5432/ -Dscalardb.jdbc.username=postgres -Dscalardb.jdbc.password=postgres
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_postgresql_14_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-oracle-18-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      oracle:
+        image: ghcr.io/scalar-labs/oracle/database:18.4.0-xe
+        credentials:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+        env:
+          ORACLE_PWD: oracle
+        ports:
+          - 1521:1521
+        options: >-
+          --health-cmd "/opt/oracle/checkDBStatus.sh"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 120
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Retrieve then store the oracle container ip address in an environment variable.
+        run: |
+          echo "oracle_db_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}'  $(docker ps -aq))" >> $GITHUB_ENV
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        #       Addressing the Oracle container with "@localhost" in the JDBC url cause the JDBC connection to fail for an unknown reason. As a workaround, using instead the container ip address is working.
+        with:
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:oracle:thin:@${{ env.oracle_db_ip }}:1521/XEPDB1 -Dscalardb.jdbc.username=SYSTEM -Dscalardb.jdbc.password=oracle
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc_oracle_18_integration_test_reports
+          path: core/build/reports/tests/integrationTestJdbc
+
+  jdbc-oracle-21-integration-test:
```

Comment:
I added it. 
For this version, I generated an image that has already prebuilt the database. This means that compared to the mint image which needs to build the database (which takes 5-10min) booting up the image now only takes a few seconds (thanks to this readme https://github.com/oracle/docker-images/tree/main/OracleDatabase/SingleInstance/samples/prebuiltdb). The drawback is that the prebuilt image size weight 15GB instead of 6GB.

The goal is to see if the Oracle integration test will take less time or not to finish by using the prebuilt image. The image will boot faster but will take more time to download eventually.

Reply:
Besides, I added a job for the latest Mysql version: 8.1